ci_env.sh

```diff
@@ -55,4 +55,6 @@ if [[ $generatedVersion != $definedVersion ]]; then
   export BUILD_TAGS="${BUILD_TAGS},experimental"
 fi
 
+echo "::set-output name=build-tags::${BUILD_TAGS}"
+
 exec "$@"
```

Comment:
Do we still need this?

Reply:
the build script in `build.yaml` still uses this as `./ci_env.sh make` (and it works o_0)